blog/2024-08-14-headlamp/index.mdx

```diff
@@ -0,0 +1,86 @@
+---
+slug: headlamp
+title: "Repost: Streamlining Kubernetes Cost Management with the New OpenCost Plugin for Headlamp"
+authors: [team]
+tags: [opencost, headlamp, kubernetes, ui, integrations]
+---
+<head>
+    <link rel="canonical" href="https://headlamp.dev/blog/2024/08/08/opencost-plugin"/>
+</head>
+
+Reposted from the [Headlamp blog](https://headlamp.dev/blog/) post: [Streamlining Kubernetes Cost Management with the New OpenCost Plugin for Headlamp](https://headlamp.dev/blog/2024/08/08/opencost-plugin), by Santhosh Nagaraj
+
+![OpenCost + Headlamp](./img/splash.jpg)
+
+[OpenCost](https://www.opencost.io/) is one of CNCF's leading open-source projects that offers real-time native cost monitoring for Kubernetes environments, providing visibility to developers and companies on the costs linked to their cloud-native applications. With OpenCost, one can trace the costs of single workloads, namespaces, and even particular labels in your Kubernetes cluster. The continuous granular cost visibility that OpenCost provides help in effectual usage of resources and better budgeting.
+
+<!--truncate-->
+
+Given how useful OpenCost is, we created a [Headlamp](https://headlamp.dev/) plugin for OpenCost to help Headlamp users track their cluster resource costs. This integration aims to solve the visibility problem between cost and cluster management, enabling developers to make more informed decisions.
```

Comment:
Would it be possible to reframe this from "we created a" to "the Headlamp team" or something along those lines?

Reply:
Yeah. I wondered about this but thought other posts we using that person as well.
I can change the version mention to something along what you suggest.